js/custom/script.js

```diff
@@ -126,4 +126,11 @@
     });
   }
 
+  // Mousedown on hidden button to trigger price recalculation when postal code input is out of focus, because
+  // we can't override it in our theme hooks.
+  if($('#edit-shipping-information-recalculate-shipping').length) {
+      $('body').on('blur', 'input.postal-code', function () {
```

Comment:
I think the issue with this (after some playing around) is that if you take slightly longer to input your postal code it will try to update too early, but if the delay is too long we end up with roughly the same result but with more complexity. The trick would be getting the delay right, but I'm not sure what that "magic number" would be.